examples/tide/Dockerfile

```diff
@@ -13,10 +13,12 @@ RUN zypper in -y gcc libopenssl-devel openssl wget && \
     chmod +x rustup-init && \
     ./rustup-init -v -y && \
     source /root/.profile && \
-    cargo build --example actix --release && \
+    cargo build --example tide --release && \
     zypper rm -y gcc libopenssl-devel && \
     rm -r rustup-init /root/.rustup
 
-CMD ['/src/target/release/examples/actix']
+COPY --from=builder /usr/src/target/release/examples/tide /sbin/
```

Comment:
I think this is missing a few steps. You need a second FROM tag for the second part. Note L24 and L2 in https://github.com/kanidm/kanidm/blob/master/kanidmd/Dockerfile :) 

did you try to build it too?

Reply:
I actually couldn't even get `docker build .` to succeed with the old actix example seemingly because only the example directory, rather than the whole repo, was being added to the container.